pykusto/expressions.py

```diff
@@ -112,16 +112,22 @@ def to_bool(self) -> 'BooleanExpression':
     def to_string(self) -> 'StringExpression':
         return StringExpression(KQL('tostring({})'.format(self.kql)))
 
-    def assign_to(self, *columns: 'Column') -> 'AssignmentBase':
+    def assign_to_single_column(self, column: 'AnyTypeColumn') -> 'AssignmentToSingleColumn':
+        return AssignmentToSingleColumn(column, self)
+
+    def assign_to_multiple_columns(self, *columns: 'AnyTypeColumn') -> 'AssignmentBase':
```

Comment:
Why do we need it here if only array can be assigned to multiple columns?

Reply:
To provide a clear error message. I once had a query fail because I was trying to assign a non-array to multiple columns, and the root cause was hard to find. Added documentation and test.